js/main.js

```diff
@@ -0,0 +1,175 @@
+/*******************************
```

Comment:
jeżeli dodasz cos do koszyka i zmienisz uklad to pojawia sie blad w konsoli
![image](https://user-images.githubusercontent.com/17024927/143682534-eaf57936-abaf-44cc-b9d6-eaf12f0dcbe2.png)

Reply:
Naprawione